website/docs/artist_tools_tray.md

```diff
@@ -0,0 +1,72 @@
+---
+id: artist_tools_tray
+title: Tray
+sidebar_label: Tray
+description: OpenPype general menu.
+---
+# Tray
+
+Click on the **OpenPype** logo to open the tray and find the major informations about your account and especially your studio files. You can customize your tray by activating options in the **Studio Settings** such ass adding **Ftrack** or **Kitsu**.
+
+![OpenPype Tray](assets/tools/tools_openpype_tray.png)
+
+## Ftrack
+*Enable it in the [Studio Settings](admin_settings_system.md).*
```

Comment:
Can't help but think that maybe this should link to the ftrack docs, and have a more generic section such as "Project management credentials"
If you enabled [Ftrack](artist_ftrack.md) or [Kitsu](artist_ftrack.md) in the [Studio Settings](admin_settings_system.md) a menu item will be present where you can login into these.

Reply:
To have a generic section, all the "Project management tools" plugins should have the same functioning. I think that as long as that's not the case, it's smarter to put one section per module.